gandalf.js

```diff
@@ -2,76 +2,77 @@
  * Extras Plugin
```

Comment:
SCRIPT-This hardcoded value might be better suited as a constant at the top of the file.